doc/modules/mod_event_pusher_http.md

```diff
@@ -33,48 +33,56 @@ It must be defined in the [`outgoing_pools` settings](../advanced-configuration/
 
 ## Options
 
-* `pool_name`: name of the pool to use (as defined in outgoing_pools)
-* `path`: path part of an URL to which a request should be sent (will be appended to the pool's prefix path).
-* `callback_module`: name of a module which should be used to check whether a notification should be sent.
+#### `modules.mod_event_pusher_http.pool_name`
+* **Syntax:** non-empty string
+* **Default:** `"http_pool"`
+* **Example:** `pool_name = "http_pool"`
+
+Name of the pool to use (as defined in outgoing_pools).
+
+#### `modules.mod_event_pusher_http.path`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `path = "/notifications"`
+
+Path part of an URL to which a request should be sent (will be appended to the pool's prefix path).
+
+#### `modules.mod_event_pusher_http.callback_module`
+* **Syntax:** string
+* **Default:** `""`
```

Comment:
Default is nothing. `callback_module = ""` would not pass validation (because it's not a module).

Reply:
Pretty much that:

```diff
diff --git a/test/config_parser_SUITE.erl b/test/config_parser_SUITE.erl
index 08a69ed98..961fd5a7e 100644
--- a/test/config_parser_SUITE.erl
+++ b/test/config_parser_SUITE.erl
@@ -1777,6 +1777,7 @@ mod_event_pusher_ivalid_opts() ->
      {http, path, 1},
      {http, callback_module, <<"wow_cool_but_missing">>},
      {http, callback_module, 1},
+     {http, callback_module, <<"">>},
      %% rabbit
      {rabbit, presence_exchange,
           #{<<"namesss">> => <<"presence">>, <<"type">> => <<"topic">>}},
```

Reply:
```suggestion
* **Default:** `"mod_event_pusher_http_defaults"`
```